mx.py

```diff
@@ -15678,7 +15696,7 @@ def processDep(dep, edge):
                     if jdk.javaCompliance < dep.jdkStandardizedSince:
                         moduleXml.element('orderEntry', attributes={'type': 'library', 'name': dep.name, 'level': 'project'})
                     else:
-                        logv("{} skipping {} for {}".format(p, dep, jdk))
```

Comment:
Why this change?

Reply:
IIRC pylint was warning about `p` not being defined at that point. But, I see it is defined in the enclosing scope. I will undo this change.